src/native/external/zlib-ng-version.txt

```diff
@@ -1,12 +1,14 @@
-v2.1.6
-74253725f884e2424a0dd8ae3f69896d5377f325
+v2.2.1
+d54e3769be0c522015b784eca2af258b1c026107
 
-https://github.com/zlib-ng/zlib-ng/releases/tag/2.1.6
+https://github.com/zlib-ng/zlib-ng/releases/tag/2.2.1
 
 We have removed the following folders from our local copy as these files are not needed for our compilation:
 
-- zlib-ng/docs/
+- zlib-ng/doc/
 - zlib-ng/test/
 - zlib-ng/arch/s390/self-hosted-builder/
 
-Apply https://github.com/dotnet/runtime/pull/105433.patch
\ No newline at end of file
+Also apply:
+- https://github.com/dotnet/runtime/commit/ecdb625035e0e3fb7c51e908713d96d2cb2080c8.patch or cherry-pick ecdb625035e0e3fb7c51e908713d96d2cb2080c8 directly.
+- https://github.com/dotnet/runtime/pull/105771/commits/d1850db0841ba80d4624df401fc89aa29ede8693.patch or cherry-pick d1850db0841ba80d4624df401fc89aa29ede8693 directly.
```

Comment:
https://github.com/dotnet/runtime/pull/105771/commits/d1850db0841ba80d4624df401fc89aa29ede8693.patch does not exist